src/System.Threading.Thread/src/System.Threading.Thread.csproj

```diff
@@ -8,8 +8,7 @@
     <AssemblyName>System.Threading.Thread</AssemblyName>
     <OutputType>Library</OutputType>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.7;uap10.1</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(TargetGroup)' != 'net463' and '$(TargetGroup)' != 'uap101aot'">netstandard1.7;uap10.1</PackageTargetFramework>
     <!-- Don't allow project reference to package dependency conversion -->
     <KeepAllProjectReferences>true</KeepAllProjectReferences>
```

Comment:
As a follow-up we can probably clean out KeepAllProjectReferences properties as well.